langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -380,6 +549,11 @@ public isolated function fromString(string s) returns xml|error = @java:Method {
 
 # Selects all the items in a sequence that are of type `xml:Text`.
 #
+# ```ballerina
+# xml books = xml `<book category="Learning"></book>Ballerina<book category="Guide"></book>Java`;
```

Comment:
Why do we need attributes?

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/9dc46b7c36c28b92b9ce360ed71621b3a6eff8c0